src/rgw/rgw_rados.cc

```diff
@@ -2847,10 +2847,12 @@ class RGWRadosThread {
 
   atomic_t down_flag;
 
+  string thread_name;
+
   virtual uint64_t interval_msec() = 0;
   virtual void stop_process() {}
 public:
-  RGWRadosThread(RGWRados *_store) : worker(NULL), cct(_store->ctx()), store(_store) {}
+  RGWRadosThread(RGWRados *_store) : worker(NULL), cct(_store->ctx()), store(_store), thread_name("radosgw") {}
```

Comment:
no, please continue :) mine will take a while to merge, let's get yours in first